src/ecosystem/corals/growth.jl

```diff
@@ -223,7 +223,7 @@ end
         dist_t::Matrix, prop_mort::SubArray{Float64})::Nothing
 
 Applies bleaching mortality by assuming critical DHW thresholds are normally distributed for
-all non-Juvenile (> 5cm diameter) size classes.
+all non-settler (> 5cm diameter) size classes.
```

Comment:
Revert this change as discussed please. Best to align with ecologists' definitions.